mediator/med_internalstate_mod.F90

```diff
@@ -263,6 +263,23 @@ subroutine med_internalstate_init(gcomp, rc)
       samegrid_atmlnd = .true.
     end if
 
+    ! flexibility to overwrite samegrid_atmlnd option
```

Comment:
@uturuncoglu Is this additional configuration required because UFS does not use an ATM mesh?

Reply:
@DeniseWorthen Yes. In CESM they are comparing names of mesh files to understand both side is using same mesh but in UFS we have no such capability.